client/webserver/site/src/js/markets.ts

```diff
@@ -785,9 +811,22 @@ export default class MarketsPage extends BasePage {
       buyBalance: 0
     }
 
+    // reset form inputs
+    const lot = '1'
+    page.lotField.setAttribute('min', lot)
+    page.lotField.setAttribute('step', lot)
+    page.lotField.value = lot
+    const lotSize = Doc.formatCoinValue(this.market.cfg.lotsize, this.market.baseUnitInfo)
+    page.qtyField.setAttribute('min', lotSize)
+    page.qtyField.setAttribute('step', lotSize)
+    page.qtyField.value = lotSize
```

Comment:
I do not agree that we should have a default here. It is not standard on any exchange's order form that I've used.

Reply:
Some reasons for why having defaults there might be a good idea:
- it provides **an example**, emphasizing the Lot concept (present in Dex, and not on most other exchanges, at least not so explicitely) = more streamlined experience for users trying out Dex for the very first time
- many first-time users will probably want to **start with 1 lot**, so less clicks overall = good for onboarding (and it doesn't affect those who want to increase this to 2, 3, ... except, as is implemented right now - one has to erase those defaults first to put in another value, this is probably a deal-breaker, but I hope this can be solved somehow)
- having those defaults there allows for `Total` calculated on page load, showing the user all the order-form parts in complete (and also happens to show min trade size which is neat)
- some manual UI testing is simpler with defaults, since I can place a bunch of limit order faster

If that's convincing enough, then I can try to address that "deal-breaker" thing, if not - I'll just remove defaults.

Reply:
Other than illustrating the lot concept, all of these points could be made about any exchange interface, yet nobody does this. I don't think it's a good idea to pre-fill critical fields for people.

Reply:
Agree.  Plus there are events that can force reload the page, and if a valid order size replaces what you had entered before, you may not notice, and end up submitting the wrong order.

Reply:
Interesting, ok - will remove.

How do you feel about slider, perhaps in the next PR ? Seems like a popular feature these days.

<img width="337" alt="image" src="https://user-images.githubusercontent.com/112318969/200044178-cdc68280-8096-4759-b98e-9fc657ffce9f.png">

Reply:
Love the slider!
I also use these buttons sometimes
![image](https://user-images.githubusercontent.com/6109680/200052207-b7ef2d1d-ccb4-45b3-852c-361a52f0061e.png)

Reply:
Haven't seen those buttons (mid, bid, ...) before, but I'll give a try to slider later on in a separate PR.